airbase/cli.py

```diff
@@ -330,14 +341,15 @@ def unverified(
     """
     asyncio.run(
         download(
-            Dataset.Verified,
+            Dataset.Unverified,
```

Comment:
😱 did we miss this before?

Reply:
Yes, unfortunately the files on the verified and unverified datasets have the same names.
Fortunately the CLI will put them on different paths by default.